PG.StarWarsGame.Files.MEG/PG.StarWarsGame.Files.MEG/Binary/File/Type/Definition/V1/MegFileNameTableRecord.cs

```diff
@@ -7,24 +10,26 @@
 using PG.Commons.Binary.File;
 using PG.Commons.Util;
 
-namespace PG.StarWarsGame.Files.MEG.Binary.File.Type.Definition
+namespace PG.StarWarsGame.Files.MEG.Binary.File.Type.Definition.V1
 {
     internal class MegFileNameTableRecord : IBinaryFile, ISizeable, IComparable
     {
-        private readonly ushort m_fileNameLength;
-        [NotNull] private readonly string m_fileName;
         private readonly Encoding m_fileNameEncoding = Encoding.ASCII;
+        private readonly ushort m_fileNameLength;
```

Comment:
m_ is an unlikely naming convention in C#.

Reply:
True, but so are a bunch used across the whole project to make the naming a tad more clear.
They are all stored in the `PetroglyphTools.sln.DotSettings` file and enforced by linter.